Tests/UnitTests/Networking/Backend/BaseBackendTest.swift

```diff
@@ -37,7 +37,14 @@ class BaseBackendTests: TestCase {
         let backendConfig = BackendConfiguration(httpClient: self.httpClient,
                                                  operationQueue: MockBackend.QueueProvider.createBackendQueue(),
                                                  dateProvider: MockDateProvider(stubbedNow: MockBackend.referenceDate))
-        self.backend = Backend(backendConfig: backendConfig, attributionFetcher: attributionFetcher)
+
+        let identity = IdentityAPI(backendConfig: backendConfig)
+        let offerings = OfferingsAPI(backendConfig: backendConfig)
+        let customer = CustomerAPI(backendConfig: backendConfig, attributionFetcher: attributionFetcher)
+        self.backend = Backend(backendConfig: backendConfig,
+                               customerAPI: customer,
+                               identityAPI: identity,
+                               offeringsAPI: offerings)
```

Comment:
I think this could use the other constructor so it doesn't manually create all the inputs?

Reply:
I was going to use the mocks here, but it turns out that isn't the right move and I forgot to switch this back to using the more friendly constructor! Thanks for catching that.